qubesadmin/tools/qvm_ls.py

```diff
@@ -479,6 +486,18 @@ def write_table(self, stream=sys.stdout):
                         table_data.append(self.get_row(vm))
                     except qubesadmin.exc.QubesVMNotFoundError:
                         continue
+                if self.sort_order in self.get_head():
+                    sort_index = self.get_head().index(self.sort_order)
```

Comment:
Store `get_head()` output to a variable to avoid repeated calls that will always have the same value.

Reply:
It is only called again once (and not in a loop). I guess we could pass this